lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -2896,6 +2896,55 @@ class DecomposeAtenMseLossOp : public OpRewritePattern<AtenMseLossOp> {
 };
 } // namespace
 
+namespace {
+class DecomposeAtenRandintLowOp : public OpRewritePattern<AtenRandintLowOp> {
+public:
+  using OpRewritePattern::OpRewritePattern;
+  LogicalResult matchAndRewrite(AtenRandintLowOp op,
+                                PatternRewriter &rewriter) const override {
+
+    Location loc = op.getLoc();
+    Type resultType = op.getType();
+    BaseTensorType resultTensorType = resultType.cast<BaseTensorType>();
+
+    int64_t cstLow, cstHigh;
+    if (!matchPattern(op.low(), m_TorchConstantInt(&cstLow)))
+      return rewriter.notifyMatchFailure(
+          op, "unimplemented: low must be a constant integer");
+    if (!matchPattern(op.high(), m_TorchConstantInt(&cstHigh)))
+      return rewriter.notifyMatchFailure(
+          op, "unimplemented: high must be a constant integer");
+
+    Value none = rewriter.create<ConstantNoneOp>(loc);
+    Value cstFalse = rewriter.create<ConstantBoolOp>(loc, false);
+    Value low = rewriter.create<Torch::ConstantFloatOp>(
+        loc, rewriter.getF64FloatAttr((double)cstLow));
+    Value high = rewriter.create<Torch::ConstantFloatOp>(
+        loc, rewriter.getF64FloatAttr((double)cstHigh));
+
+    BaseTensorType floatResultType =
+        resultTensorType
+            .getWithSizesAndDtype(resultTensorType.getSizes(),
+                                  rewriter.getF64Type())
+            .cast<BaseTensorType>();
+    Value emptyTensor = rewriter.create<AtenEmptyMemoryFormatOp>(
+        loc, floatResultType, op.size(), /*dtype=*/none, /*layout=*/none,
```

Comment:
Either the `dtype` argument needs to be specified (default dtype is f32 but result type is f64), or the result type should be changed to `f32`.